test_suites/h264/JVT-AVC_v1.json

```diff
@@ -1,7 +1,7 @@
 {
-    "name": "JVT-AVC_V1",
+    "name": "JVT-AVC_v1",
```

Comment:
This change broke the BC.

Reply:
It's true, PR to fix:

https://github.com/fluendo/fluster/pull/202

Reply:
Sorry, my thinking didn't go that far.
So the names of test suites should be fixed until we make a new major release?
I guess new test suites should not be added to the package without incrementing the versions, right?

Reply:
+1
We need start doing fluster releases.

Reply:
[https://github.com/fluendo/fluster/pull/202](https://github.com/fluendo/fluster/pull/202) merged